src/nvim/syntax.c

```diff
@@ -994,11 +994,10 @@ void syn_stack_free_all(synblock_T *block)
  */
 static void syn_stack_alloc(void)
 {
-  long len;
   synstate_T *to, *from;
   synstate_T *sstp;
 
-  len = syn_buf->b_ml.ml_line_count / SST_DIST + Rows * 2;
+  int len = syn_buf->b_ml.ml_line_count / SST_DIST + Rows * 2;
```

Comment:
Is it safe to assume `int` is at least 32 bits?

Reply:
yes